functions/util/messageWriters.ts

```diff
@@ -0,0 +1,63 @@
+import { Event, Slot } from "../../webapp/src/util/types";
+
+type TimeFormat = {
+    shortTimezone: string,
+    locale: string,
+    timezone: string,
+}
+
+export const timeFormats: {[key: string] : TimeFormat} = {
+    GMT: {
+        shortTimezone: "GMT",
+        locale: "en-GB",
+        timezone: "Europe/London",
+    },
+    AU: {
+        shortTimezone: "AU",
+        locale: "en-AU",
+        timezone: "Australia/Sydney",
+    },
+    EST: {
+        shortTimezone: "EST",
+        locale: "en-US",
+        timezone: "America/New_York",
+    },
+    CST: {
+        shortTimezone: "CST",
+        locale: "en-US",
+        timezone: "America/Chicago",
+    },
+    MST: {
+        shortTimezone: "MST",
+        locale: "en-US",
+        timezone: "America/Denver",
+    },
+    PST: {
+        shortTimezone: "PST",
+        locale: "en-US",
+        timezone: "America/Los_Angeles",
+    },
+};
+
+export const getLineupText = (event: Event, timeFormat: TimeFormat): string => {
```

Comment:
We'll want to keep those since it's still used to process the legacy whiteboard messages.  Once that's removed, old message generators should be removed.